templates/webapi/test/overview.html.ep

```diff
@@ -231,10 +235,18 @@
                         <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
                     </div>
                     <div id="add-comments-controls">
-                        <button type="submit" class="btn btn-danger">
-                            <i class="fa fa-comment"></i> Submit comment on all <%= scalar @$job_ids %> jobs
-                        </button>
-                        <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Discard</button>
+                      <button type="submit" name="bulkAction" class="btn btn-warning"
+                              value="restartAndCommentJobs"
```

Comment:
```suggestion
                              value="restartOrCommentJobs"
```

Reply:
Are we in the same page here? we have two button. one which will trigger restart with a comment and one that just post comments.

Reply:
Right, sorry. Keep it here. Only the general dialog with the "undo icon in comment bubble" button should be called "restartOrCommentJobs"